### PR TITLE
Allows for opening app link in new tab

### PR DESCRIPTION
**Pull request by drzackyll**

### CODE REVIEW

File: etools-app-selector.html [hunk 1]
```diff
@@ -129,11 +129,11 @@
               <template is="dom-repeat" items="[[menuOptions]]" as="app">
                 <div class="panel-row">
                   <div class="content-wrapper panel">
-                    <a on-tap="goToPage">
+                    <a on-tap="goToPage" href="[[baseSite]]/[[app.url]]">
```
Comment (marko911) [inline etools-app-selector.html]: But what is baseSite @drzackyll  ? Remember that it was used specifically in the other element you got it from so you must declare it in this element and assign it the proper value. This is a standalone element so assume it knows nothing outside of the properties it has or ones passed to it.

**Comment by drzackyll** [comment]
@marko911 updates pushed

File: etools-app-selector.html [hunk 2]
```diff
@@ -226,9 +240,11 @@
        *  `app.url="admin" will change location to 'http://myBaseUrlAndPort/admin/' `
        *
        */
+
       goToPage: function(e) {
-        var app = e.model.app;
```
Comment (marko911) [inline etools-app-selector.html]: Revert to this.

**Comment by drzackyll** [comment]
@marko911 pushed latest changes; will need to add new style variable --app-selector-text-color to dashboard/app-theme.html

**Pull request merged**